man/systemd.link.xml

```diff
@@ -133,7 +133,7 @@
         <term><varname>Type=</varname></term>
         <listitem>
           <para>A whitespace-separated list of shell-style globs matching the device type, as exposed by
-          <command>networkctl status</command>. If the list is prefixed with a "!", the test is inverted.
+          <command>networkctl list</command>. If the list is prefixed with a "!", the test is inverted.
```

Comment:
it does actually show this info, if you specify the iface name, which I guess is meant here. That said, using "networkctl list" probably is a better choice, since it requires no extra argument and dumps the whole table easily parsable.

So patch looks good, but could you maybe slightly alter the commit msg, saying he "networkctl status" might not be wrong (as long as you specify the iface name), but "network list" a better choice, since it always shows the necessary info.

Reply:
I revised the commit log to clarify this.